sdk/src/main/java/io/opentelemetry/sdk/trace/Samplers.java

```diff
@@ -233,4 +244,33 @@ public boolean isSampled() {
       return Collections.emptyMap();
     }
   }
+
+  /** Probability-based sampling decision with a single attribute for the probability. */
+  @Immutable
+  @AutoValue
+  abstract static class ProbabilityDecision implements Decision {
+
+    ProbabilityDecision() {}
+
+    /**
+     * Creates sampling decision without attributes.
+     *
+     * @param decision sampling decision
+     * @param probability the probability that was used for the decision.
+     */
+    static ProbabilityDecision create(boolean decision, double probability) {
+      return new AutoValue_Samplers_ProbabilityDecision(
+          decision, singletonMap("samplingProbability", doubleAttributeValue(probability)));
```

Comment:
All other semantic conventions use a prefix followed by a dot and underscores instead of camelCase (e.g. `messaging.protocol_version`). So a better name would probably be `probability_sampler.probability`.

Reply:
Or just `sampler.probability` if we intend to have this cross samplers.

Reply:
yeah, I was just throwing something in there to start this discussion!  Agree this should be something in the semantic conventions. Should we put this PR on hold while that gets done? @pavolloffay as the original requester, can you create an issue in the spec repo and get the ball rolling?

Reply:
I went ahead and put in a PR with `sampling.probability` as the attribute name. https://github.com/open-telemetry/opentelemetry-specification/pull/570.